application/mcp/rest_client.py

```
@@ -0,0 +1,318 @@
+        response = self._session.get(
+            url,
+            params=_encode_query(query_params),
+            timeout=self._timeout,
+            allow_redirects=False,
+        )
+        # Never retain cookies across tool calls (no credential forwarding).
+        if hasattr(self._session, "cookies"):
+            try:
+                self._session.cookies.clear()
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)